bcl/src/bc_led.c

```diff
@@ -79,15 +130,17 @@ void bc_led_set_mode(bc_led_t *self, bc_led_mode_t mode)
     {
         case BC_LED_MODE_TOGGLE:
         {
-            if (pattern == 0x00000000)
+            if (pattern == 0)
             {
                 self->_pattern = 0xffffffff;
+                self->_count = 0;
```

Comment:
maybe move all self->_count = 0; before the switch

Reply:
I have checked that possibility and we cannot do that - we can apply `_count = 0` only for the steady modes (fully on/fully off), but we don't want to do that for the blinking patterns.